src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -34640,6 +34640,519 @@ .maxstack  2
                 """);
         }
 
+        [Fact]
+        public void List_SingleSpread_CustomCollection_NotICollectionAndNoStructEnumerator()
+        {
+            var source = """
+                using System.Collections;
+                using System.Collections.Generic;
+
+                class MyCollection(List<int> list) : IEnumerable<int>
+                {
+                    public IEnumerator<int> GetEnumerator() => list.GetEnumerator();
+
+                    IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+                }
+
+                class C
+                {
+                    static void Main()
+                    {
+                        M(new([1, 2, 3])).Report();
+                    }
+
+                    static List<int> M(MyCollection c) => [..c];
+                }
+                """;
+
+            var verifier = CompileAndVerify([source, s_collectionExtensionsWithSpan], expectedOutput: IncludeExpectedOutput("[1, 2, 3],"), targetFramework: TargetFramework.Net80, verify: Verification.Skipped);
```

Comment:
>s_collectionExtensionsWithSpan [](http://example.com/codeflow?start=53&length=30)

It looks like all new tests can use `s_collectionExtensions` rather than `s_collectionExtensionsWithSpan`.